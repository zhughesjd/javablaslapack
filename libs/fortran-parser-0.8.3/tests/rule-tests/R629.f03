
! Testing allocate_object_list
deallocate (this,that,theother)
end
