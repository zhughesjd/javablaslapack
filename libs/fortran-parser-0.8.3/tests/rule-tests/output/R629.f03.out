1101:begin
629:list-begin
613 this false false
612 1
613 that false false
612 1
613 theother false false
612 1
629:list 3
1103
1101 false true false
